packages/twenty-front/src/config/index.ts

```diff
@@ -28,3 +28,12 @@ export const REACT_APP_SERVER_BASE_URL =
   window._env_?.REACT_APP_SERVER_BASE_URL ||
   process.env.REACT_APP_SERVER_BASE_URL ||
   getDefaultUrl();
+
+export const SENTRY_RELEASE =
+  window._env_?.SENTRY_RELEASE ||
+  process.env.SENTRY_RELEASE;
```

Comment:
I would actually pass these through ClientConfig alongside with SENTRY_DSN to be consistent and not add more env variables on frontend side